openreview/venue_request/process/revisionProcess.py

```diff
@@ -624,6 +624,23 @@ def process(client, note, invitation):
         client.post_invitation(submission_revision_invitation)
 
         print('Conference: ', conference.get_id())
+        comment_note = openreview.Note(
```

Comment:
@racheljsmart this code must be in the process function of your venue request invitations.

Reply:
Maybe the comment could read "The {invitation_type.replace("_", " ")} process has been completed." ?

Currently "The process for the {invitation_type.replace("_", " ")} process has been completed." is a little redundant.